contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -138,17 +137,17 @@ The `token-id` must be 6 characters and the `token-secret` must be 16 characters
 
 #### NEW: Bootstrap Token Secrets
 
-Bootstrap tokens are stored and managed via Kubernetes secrets in the `kube-system` namespace.  They have type `bootstrap.kubernetes.io/token`.
+Bootstrap tokens are stored and managed via Kubernetes secrets in the `kube-system` namespace.  They have the type `bootstrap.kubernetes.io/token`.
 
 The following keys are on the secret data:
 * **token-id**. As defined above.
 * **token-secret**. As defined above.
 * **expiration**. After this time the token should be automatically deleted.  This is encoded as an absolute UTC time using RFC3339.
 * **usage-bootstrap-signing**. Set to `true` to indicate this token should be used for signing bootstrap configs.  If this is missing from the token secret or set to any other value, the usage is not allowed.
-* **usage-bootstrap-authentication**. Set to true to indicate that this token should be used for authenticating to the API server.  If this is missing from the token secret or set to any other value, the usage is not allowed.  The bootstrap token authenticagtor will use this token to auth as a user that is `system:bootstrap:<token-id>` in the group `system:bootstrappers`.
+* **usage-bootstrap-authentication**. Set to true to indicate that this token should be used for authenticating to the API server.  If this is missing from the token secret or set to any other value, the usage is not allowed.  The bootstrap token authenticator will use this token to auth as a user that is `system:bootstrap:<token-id>` in the group `system:bootstrappers`.
 * **description**. An optional free form description field for denoting the purpose of the token.  If users have especially complex token management neads, they are encouraged to use labels and annotations instead of packing machined readable data in to this field.
+* **auth-groups**. A comma-separated list of which groups the token should be authenticated as. All groups must have the `system:bootstrappers:` prefix.
```

Comment:
It seems like it makes sense if the usage of these token's is going to expand past simple CSR use cases. I'd be in favor of putting on this for now. It doesn't seem like it'd be necessary for beta.